### PR TITLE
KG-514

### PR DESCRIPTION
**Pull request by davidizac**
G-Suite plugin

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        "name":"CLIENT_ID",
+        "viewName":"Client Id",
+        "type":"string"
+      }
+    ],
+    "methods": [
+    {
+        "name": "createUser",
+        "viewName": "Create a user",
+        "params": [
```
Comment (kadosh1000) [inline src/config.json:35]: Please add a parameter for `changePasswordAtNextLogin` of type boolean.

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                return reject(err)
+            }
+            resolve()
+        });
+    })
+}
+
+function addUserToGroup(action) {
+    const auth = _authorize(action);
+    const service = google.admin({ version: 'directory_v1', auth });
```
Comment (kadosh1000) [inline src/app.js]: @DavidIzaac this is not correct any more

File: src/app.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    })
+}
+
+function _addUserToGroup(service, groupKey, user) {
+    return new Promise((resolve,reject) => {
+        return service.members.insert({ groupKey: groupKey, resource: user },(err,res) => {
+            if(err){
+                return reject(err)
+            }
+            resolve()
```
Comment (kadosh1000) [inline src/app.js]: @DavidIzaac resolve with the res you've got from the insert

File: src/app.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+
+const { google } = require('googleapis');
+
+
+function _authorize(action,settings) {
+    let client_id = settings.CLIENT_ID || action.params.CLIENT_ID;
+    let client_secret = settings.CLIENT_SECRET || action.params.CLIENT_SECRET;
+    let redirect_uris = ["urn:ietf:wg:oauth:2.0:oob", "http://localhost"];
```
Comment (Or-Sharbat) [inline src/app.js:8]: good job :)

**Pull request merged**